python/dgl/graphbolt/subgraph_sampler.py

```diff
@@ -0,0 +1,18 @@
+"""Subgraph samplers"""
+
+from torchdata.datapipes.iter import IterDataPipe
+
+
+class MockSubgraphSampler(IterDataPipe):
+    """A mock subgraph sampler."""
+
+    def __init__(self, dp, graph, sampler):
```

Comment:
what's the definition of sampler here?

Reply:
It is probably better to pass in a sample_func such that users can define it outside.